response_operations_ui/controllers/sample_controllers.py

```diff
@@ -33,3 +33,20 @@ def upload_sample(short_name, period, file):
                  sample_id=sample_summary['id'])
 
     return sample_summary
+
+
+def get_sample_summary(summary_uuid):
+    logger.debug('Getting sample summary', summary_uuid=summary_uuid)
```

Comment:
Maybe use structlog's wrapped logger here? Not sure if this is a pattern in RAS or not... but it's used in SDX and EQ.

Reply:
Frontstage uses it in most of its modules.

Reply:
This is using the wrapped logger?
line 11 of this file `logger = wrap_logger(logging.getLogger(__name__))`

Reply:
Ah that makes it an easier change then 👍 cheers @andrewmil